usaspending_api/conftest.py

```diff
@@ -67,6 +72,23 @@ def add_view_protection():
         pass
 
 
+def create_download_views():
+    """
+    In some cases the download views are based off of either Tables or Materialized Views that are converted to
+    Views for ease of use with PyTest. In these cases the download view is dropped prior to creation of the base
+    Table/Materialized View/View. This function ensures that all download Views exist in case they were dropped.
+    """
+    try:
+        download_view_dir = Path("usaspending_api/download/sql")
+        download_view_files = [download_view_dir / file_name for file_name in os.listdir(download_view_dir)]
+        sql = ""
+        for view in download_view_files:
```

Comment:
Should the exception handling take place within the for loop here, so if one view already exists and throws and exception, later views will still be created?

Reply:
It certainly could, but it could also be removed in hindsight. The SQL that it is running `DROP IF EXISTS` the view and then `CREATE`.

Reply:
I'm good with either approach